CHANGES.md

```diff
@@ -2,6 +2,9 @@
 
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.213 Aug 01 2022
+- Add `label` locator to labels resource and added `Delete` method to label resource.
```

Comment:
Other `CHANGES.md` items with multiple changes list them on different lines instead of using `and` to conjoin them in one.  Consider splitting it up into two lines:

```
- Add `label` locator to labels resource
- Add `Delete` method to label reosurce
```